models/pod.go

```diff
@@ -145,6 +146,14 @@ func NewPod(
 		pod.ResourcesRequestsMemory = requests.Memory
 	}
 	err := pod.configureHostPorts(clientset, redisClient)
+
+	if err == nil {
+		reporters.GetInstance().Report("gru.new", map[string]string{
+			"name":      game,
+			"scheduler": namespace,
```

Comment:
Shouldn't this receive a reason?

Reply:
Nope. It only matters why a pod was deleted, creations are all put together in the same metric.